Python/Product/PythonTools/PythonTools/Debugger/DebugAdapterLauncher.cs

```diff
@@ -56,7 +56,8 @@ public sealed class DebugAdapterLauncher : IAdapterLauncher {
                 JObject obj = new JObject()
                 {
                     ["remote"] = uri,
-                    ["options"] = "SHOW_RETURN_VALUE={0}".FormatInvariant(debugService.ShowFunctionReturnValue)
+                    ["options"] = "SHOW_RETURN_VALUE={0};BREAK_SYSTEMEXIT_ZERO={1}"
+                        .FormatInvariant(debugService.ShowFunctionReturnValue, debugService.BreakOnSystemExitZero)
```

Comment:
If you watch to switch to the (newer and cleaner) `debugOptions` API:
https://github.com/microsoft/ptvsd/blob/9666eac569bb170d191896c5feb083d0a6ff1bf4/README.md#launch-request-arguments

Reply:
In terms of compatibility with older versions of ptvsd, this has been around for over a year:
https://github.com/microsoft/ptvsd/issues/245

Reply:
Updated to use new API